### PR TITLE
feat(request): logging 'join' and 'leave'

### PR DESCRIPTION
**Pull request by mwegman**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)
SPARK-152947 #code-complete

## Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Test Coverage

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* SDK Version
* Node/Browser Version
* NPM Version

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 1]
```diff
@@ -43,6 +43,11 @@ export default class MeetingRequest extends StatelessWebexPlugin {
       sipUri, deviceUrl, locusUrl, resourceId, correlationId, ensureConversation, moderator, pin, moveToResource, roapMessage
     } = options;
 
+    LoggerProxy.logger.log(
+      `${MeetingRequest.name}#${this.joinMeeting.name} --> Joining a meeting`,
```
Comment (mwegman) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:47]: I thought this would be less brittle than what we were doing. I can change it back if we would rather

**Comment by arun3528** [comment]
@mwegman  do you have a screen shot on how is shows up in logs . 

Just want to check if it has something like . Join-initiated, join-success, join-failed for the api call

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js [hunk 2]
```diff
@@ -165,21 +170,29 @@ export default class MeetingRequest extends StatelessWebexPlugin {
 
   /**
    * Make a network request to leave a meeting
-   * @param {Object} options
-   * @param {String} options.deviceUrl
-   * @param {String} options.resourceId,
-   * @param {String} options.correlationId
+   * @param {Url} locusUrl
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:173]: (we discussed this on call but leaving here for posterity) Let's add the `options.` back to show they are destructured params.

File: packages/node_modules/@webex/plugin-meetings/src/meeting/request.js (hunk 1 above)
Comment (adamweeks) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/request.js:47]: I like this! Want to change all the others in the plugin too? 😉

**Comment by mwegman** [comment]
Screen Shot 2020-06-22 at 10 22 00 AM
Screen Shot 2020-06-22 at 10 21 43 AM

**Comment by mwegman** [comment]
Added 'grouping' to the logger for 'join' and 'leave'

File: packages/node_modules/@webex/plugin-logger/src/logger.js [hunk 3]
```diff
@@ -17,12 +17,14 @@ import {
 
 const precedence = {
   silent: 0,
-  error: 1,
-  warn: 2,
-  log: 3,
-  info: 4,
-  debug: 5,
-  trace: 6
+  group: 1,
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-logger/src/logger.js:20]: Not sure what is going on here, is this needed as part of the PR? Could we make it a separate commit with `logger` being the scope?

File: packages/node_modules/@webex/plugin-meetings/src/meeting/index.js [hunk 4]
```diff
@@ -2020,6 +2020,9 @@ export default class Meeting extends StatelessWebexPlugin {
       });
     }
 
+    LoggerProxy.logger.group('JOIN MEETING');
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/index.js:2023]: Surprised the logger proxy unit tests aren't failing without this being mocked

**Comment by adamweeks** [comment]
@mwegman should we close this?
